control-plane/cmd/post-install/kafka_source_dispatcher_deleter.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/system"
+)
+
+func (d *kafkaSourceDeleter) DeleteDispatcher(ctx context.Context) error {
+	// Delete deployment.apps/kafka-source-dispatcher
+	const sourceDispatcherDeploymentName = "kafka-source-dispatcher"
```

Comment:
RBAC is set for this resource to be `delete`d, right?

Reply:
I see it is here: https://github.com/knative-sandbox/eventing-kafka-broker/pull/2219/files#diff-7ae2415f2a98a495c12f118fd6eac7ff61a6a16781a0da32809e41e0010caafcR93

:+1:

Reply:
https://github.com/knative-sandbox/eventing-kafka-broker/pull/2219/files#diff-7ae2415f2a98a495c12f118fd6eac7ff61a6a16781a0da32809e41e0010caafcR93